db/schema.rb

```diff
@@ -54,12 +54,14 @@
   end
 
   create_table "deploy_groups", force: :cascade do |t|
-    t.string   "name",           limit: 255, null: false
-    t.integer  "environment_id", limit: 4,   null: false
+    t.string   "name",                  limit: 255, null: false
+    t.integer  "environment_id",        limit: 4,   null: false
     t.datetime "deleted_at"
-    t.datetime "created_at",                 null: false
-    t.datetime "updated_at",                 null: false
-    t.string   "env_value",      limit: 255, null: false
+    t.datetime "created_at",                        null: false
+    t.datetime "updated_at",                        null: false
+    t.string   "env_value",             limit: 255, null: false
+    t.integer  "kubernetes_cluster_id", limit: 4
```

Comment:
I'd kind of rather see a join table with a unique deploy group id index to keep deploy groups from knowing anything about kubernetes. You should be able to keep everything in a plugin then as well.